src/cli/dispatchers/messages.py

```
@@ -289,11 +316,15 @@ def _clear_preview(streaming_state: dict) -> None:
             streaming_state["chunks"] = []
 
     def _finalize_streaming(
-        self, streaming_state: dict, status, rendered_messages: set[str]
+        self,
+        streaming_state: dict,
+        status,
+        rendered_messages: set[str],
+        stop_status: bool = True,
```

Ruff (0.14.7)

Boolean-typed positional argument in function definition

(FBT001)